try:
    result = 10/0
except Exception as error:
    print("error",error)